__tests__/app.test.js

```diff
@@ -357,3 +356,43 @@ describe("GET /api/users", () => {
   });
 });
 
+describe("GET /api/articles (sorting queries)", () => {
```

Comment:
This doesnt need to be a separate describe as it is build up from the original endpoint so it can be combined with those tests